### PR TITLE
Fractional symbols added

### PR DESCRIPTION
**Pull request by parthpanchal123**
Fractional symbols added .

```
½ \frac12
⅓ \frac13
⅔ \frac23
¼ \frac14
¾ \frac34
⅕ \frac15
⅖ \frac25
⅗ \frac35
⅘ \frac45
⅙ \frac16
⅚ \frac56
⅛ \frac18
⅜ \frac38
⅝ \frac58
⅞ \frac78

```

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Currently there is a space and a tab in between and 2 tabs at the end of each line

There should be only 1 tab between and 1 tab at the end for it to work

Let me know if I can help

**Comment by parthpanchal123** [comment]
> Currently there is a space and a tab in between and 2 tabs at the end of each line
> 
> There should be only 1 tab between and 1 tab at the end for it to work
> 
> Let me know if I can help

The Tab is 4 spaces, right ? Mine was set to 8 spaces, hence I got the issue. Fixing it!

**Comment by DenverCoder1** [comment]
It should not have any spaces, just a single tab character. If your editor inserts spaces, use a different one or copy-paste the tabs from a different line.

File: dictionary.txt [hunk 1]
```diff
@@ -881,4 +881,19 @@ _rp	₎
...
+\frac15	⅕				
+\frac25	⅖				
+\frac35	⅗				
+\frac45	⅘				
+\frac16	⅙				
+\frac56	⅚				
+\frac18	⅛				
+\frac38	⅜					
+\frac58	⅝				
+\frac78	⅞
```
Comment (DenverCoder1) [inline dictionary.txt:898]: Too many tabs at the ends of the lines

```suggestion
\frac12	½	
\frac13	⅓	
\frac23	⅔	
\frac14	¼	
\frac34	¾	
\frac15	⅕	
\frac25	⅖	
\frac35	⅗	
\frac45	⅘	
\frac16	⅙	
\frac56	⅚	
\frac18	⅛	
\frac38	⅜	
\frac58	⅝	
\frac78	⅞	
```

**Review by DenverCoder1 — APPROVED** [review]
Looks good! Thanks for contributing! :tada:

**Pull request merged**